config/dev.exs

```diff
@@ -8,7 +8,7 @@ config :trento, Trento.Repo,
   hostname: "localhost",
   port: 5433,
   show_sensitive_data_on_connection_error: true,
-  pool_size: 10,
+  pool_size: 5,
```

Comment:
The pool size is too large for the container we use in development so I reduced it